.github/workflows/packages.yml

```diff
@@ -188,7 +188,7 @@ jobs:
         done
 
         if [ ! -z "$packages" ]; then
-          ./scripts/run-docker.sh ./build-package.sh -I -a ${{ matrix.target_arch }} $packages
+          ./scripts/run-docker.sh ./build-package.sh -I -a ${{ matrix.target_arch }} --multixz $packages
```

Comment:
This should not be done by default, if you are aiming for safety.

Reply:
But then most builds are carried out on Github Actions and I think they impose the time limit...

Reply:
Is the time limit significant? I would say it is negligible in most cases. At most only a handful of packages will really need the multi-threading.

Reply:
And for such a package it should be sufficient to specify the flag in its `build.sh`. This change in the workflow is not necessary at all.

Reply:
Benchmark with `libllvm`:

Size in MB:
| Arch | Time with `--multixz` | Size with `--multixz` | without `--multixz` | Size without `--multixz` |
| --- | --- | --- | --- | --- |
| aarch64 | 2:39 | 144 | 2:39 | 143 |
| arm | 2:40 | 148 | 3:38 | 146 |
| i686 | 2:41 | 162 | 3:47 | 160 |
| x86_64 | 2:46 | 18 | 2:40 | 156 |